cmdline.c

```
@@ -127,8 +157,18 @@ int parse_cmd_line(int argc, char *argv[], struct endpoint *A,
 
 			case 'o':
 				/* save capture onto the output files */
-				out_filenames[0] = DEFAULT_A_TO_B_DUMPFILENAME;
-				out_filenames[1] = DEFAULT_B_TO_A_DUMPFILENAME;
+				if (parse_output_filenames("", out_filenames) != 0) {
+					fprintf(stderr, "Invalid output filenames prefix.\n");
+					return ret;
+				}
+				break;
+
+			case 'f':
+				/* add output files prefix */
+				if (parse_output_filenames(optarg, out_filenames) != 0) {
+					fprintf(stderr, "Invalid output filenames prefix.\n");
+					return ret;
+				}
```

I would raise an error somewhere saying that passing `-o` *and* `-f` is incorrect. Both enable the dump but depending on the order of the flags in the command line you may end up using the default fnames with or without the prefix.

The check at least prevents unwanted surprises.

---

Ok, they are now incompatible options